React/Views/RefreshControl/RCTRefreshControl.m

```diff
@@ -108,6 +110,19 @@ - (void)endRefreshingProgrammatically
   }
 }
 
+- (void)applyProgressViewOffset
+{
+  // progressViewOffset must be converted from the ScrollView parent's coordinate space to
+  // the coordinate space of the RefreshControl. This ensures that the control respects any
+  // offset in the view hierarchy, and that progressViewOffset is not inadvertently applied
+  // multiple times.
+  UIView *scrollView = self.superview;
+  UIView *target = scrollView == nil ? nil : scrollView.superview;
```

Comment:
Here you should be able to rewrite using optional chaining: 

```js
UIView *target = scrollView?.superview;
```

Reply:
Oh I just realized my feedback doesn't apply (this is objective-C). 😬

Reply:
I was just going to say that.